RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -72,11 +72,17 @@ public bool Generate(NuGetReference analyzeRef, string language, string sqaleFil
             }
             SupportedLanguages.ThrowIfNotSupported(language);
 
-            string nuGetDirectory = Utilities.CreateTempDirectory(".nuget");
-            IPackage package = this.packageHandler.FetchPackage(analyzeRef.PackageId, analyzeRef.Version, nuGetDirectory);
+            IPackage package = this.packageHandler.FetchPackage(analyzeRef.PackageId, analyzeRef.Version);
 
             if (package != null)
             {
+                // Build machines will need to install this package, it is not feasible to create plugins for packages requiring license acceptance
+                if (PackageRequiresLicenseAcceptance(package)) 
+                {
+                    this.logger.LogError(UIResources.APG_NGPackageRequiresLicenseAcceptance);
```

Comment:
This brings up a thought - is it wise to return early from this recursive search or should we complete the tree, showing an error message for each package that requires a license acceptance?